### PR TITLE
Update paper, reword NND and add references

### PR DESCRIPTION
**Pull request by padster**

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Additionally, if a record of stimulus or trigger times is provided `pyNeuroTrace
...
...rewritten to be almost entirely vectorized in their calculations. The benefit is more noticeable when comparing the difference in performance while using large data sets, such as those generated using a longer time series or faster acquisition rates. These vectorized implementations gain further speed by being executed on a GPU using the Cupy Python library [@cupy_learningsys2017]. The GPU-accelerated filters can be imported from the `pyneurotrace.gpu.filters` module, and a CUDA-compatible graphics card is required for their execution. This functionality is becoming increasingly crucial as acquisition rates increase for kilohertz imaging of activity [@Zhang2019], which can generate arrays of hundreds of thousands of data points in just a few minutes of recording. \autoref{fig:CPUvsGPU} shows the difference in calculating arrays of various sizes using either the CPU or vectorized GPU-based approach of the $\Delta F/F$ function. The CPU used in these calculations was an Intel i5-9600K with six 4.600GHz cores; the GPU was an NVIDIA GeForce RTX 4070 with CUDA Version 12.3. When compared, the GPU calculation was on average TODO times faster on time series up to 1 million points in size.
```
Comment (padster) [inline paper/paper.md:111]: @Peter-Hogg I left this as `TODO`, once merged are you able to replace it with the actual average speed difference? visually looks like 100 or so.

File: paper/paper.md [hunk 2]
```diff
@@ -84,22 +84,22 @@ $$
...
+The event detection module uses several strategies to identify neuronal activity events in time series datasets. These methodologies have been previously discussed and compared by Sakaki *et al.* [@Sakaki2018]. These include two generalizable methods and one that requires prior knowledge of recorded event shape. The generalizable methods include filtering the signal through an exponentially weighted moving average (ewma) or cumulative sum of movement above the mean (cusum). The final filter is a matched filter that finds the probability of the trace matching a previously defined shape, such as one described by an exponential rise and decay of calcium signal generated by a GECI. Each algorithm was chosen as they are is computationally efficient, so are able to be run in real-time during an experiment if required, or process very large time series.
```
Comment (Peter-Hogg) [inline paper/paper.md:93]: Computationally efficient algorithms were selected, allowing for the possibility of applying them in real-time during experiments.

File: paper/paper.md (hunk 1 above)
Comment (Peter-Hogg) [reply paper/paper.md:111]: the mean was 103.5 times faster when comparing between the two lines, the max is about 150 and that's for 100,000 not 1 million

Comment (padster) [reply paper/paper.md:111]: Thanks, updated to 'in the order of 100 times faster' and changed to 100,000

**Pull request merged**